core/src/avm2/globals/flash/events/PressAndTapGestureEvent.as

```diff
@@ -0,0 +1,35 @@
+package flash.events
+{
+    public class PressAndTapGestureEvent extends GestureEvent
+    {
+        public static const GESTURE_PRESS_AND_TAP : String = "gesturePressAndTap";
+
+        public var tapLocalX: Number;
```

Comment:
These should be getters/setters to match Flash

Reply:
Ok I can change theses.
By the way `localX`and `localY` are defined the same way in `GestureEvent`.